server/migrations/usersMigration.js

```diff
@@ -0,0 +1,44 @@
+import { MongoClient, ObjectId } from 'mongodb'
```

Comment:
what do you think about standardizing on a file name structure for migrations? I'm thinking we could prefix a timestamp to the file name, maybe something like `2022-12-06-03-30-migrate-users.js`? That way we have a sense of when the migration was created and what we may or may not need to run.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237/commits/e1e17c4086079501bec2e0c2cdfe453e26b16af5